lib/sufia/engine.rb

```diff
@@ -23,6 +23,9 @@ class Engine < ::Rails::Engine
     initializer 'requires' do
       require 'activerecord-import'
       require 'hydra/derivatives'
+      require 'sufia/analytics'
```

Comment:
@jcoyne , @mjgiarlo , in rails console you can check if you get good data for Sufia::Analytics.Profile.sufia__download

I get data only when I add the three lines in engine.rb ( I am not saying that there might not be another way) but this is what is working for me and in Sufia 6 (https://github.com/projecthydra/sufia/blob/6.x-stable/sufia-models/lib/sufia/models/engine.rb#L71)